frontend/src/views/SecretOverviewPage/components/SelectionPanel/SelectionPanel.tsx

```diff
@@ -68,6 +69,16 @@ export const SelectionPanel = ({
     )
   );
 
+  const getDeleteModalTitle = () => {
+    if (selectedFolderCount > 0 && selectedKeysCount > 0) {
+      return "Do you want to delete the selected secrets and folders across envs?";
+    } else if (selectedFolderCount > 0 && selectedKeysCount === 0) {
+      return "Do you want to delete the selected folders across envs?";
```

Comment:
change `envs` to environments for all of the tests

Reply:
made the update